charts/latest-v2/azuredisk-csi-driver/values.yaml

```diff
@@ -74,6 +74,9 @@ controller:
   provisionerWorkerThreads: 40
   attacherWorkerThreads: 500
   vmssCacheTTLInSeconds: -1
+  enableAzureClientAttachDetachRateLimiter: false
+  azureClientAttachDetachRateLimiterQPS: 2.6
```

Comment:
Does this mean our testing region allow 468 queries per 3 minutes?

Reply:
That is the increased limit in some regions but not the one we are currently testing

Reply:
Yes, and will keep these values to defaults.